pytorch_toolkit/instance_segmentation/configs/instance-segmentation-0228.py

```diff
@@ -0,0 +1,85 @@
+_base_ = '../../../external/mmdetection/configs/mask_rcnn/mask_rcnn_r50_caffe_fpn_mstrain-poly_3x_coco.py'
+model = dict(
+    pretrained='open-mmlab://detectron2/resnet101_caffe',
+    backbone=dict(
+        depth=101),
+    neck=dict(
+        out_channels=128),
+    rpn_head=dict(
+        in_channels=128,
+        feat_channels=128),
+    roi_head=dict(
+        bbox_roi_extractor=dict(
+            out_channels=128),
+        bbox_head=dict(
+            in_channels=128,
+            fc_out_channels=512,
+            roi_feat_size=7),
+        mask_roi_extractor=dict(
+            roi_layer=dict(type='RoIAlign', out_size=7, sample_num=0),
```

Comment:
1. What's the quality degradation at the OpenVINO side?
1. How does mmdetection RoIAlign compare with torchvision one at the PyTorch & OpenVINO sides?
1. Is it the aligned version?

Reply:
1. Metrics are the same in `Pytorch` and `OpenVINO` for first three models (at least in v.2020.4) when all images are resized to a target resolution without keeping ratio of width and height. For the two last models there is a quality degradation because of image preprocessing where is set to True `keep_ratio` option and some images are stretched by height when OpenVINO model expects horizontal images. I forgot about it.
  2. I do not know, did not try to change the type. It just works without any issues. 
  3. What do you mean?

Reply:
1. Cool. It's not what I would expect though.